### DISCUSSION TITLE
victor 86b

### DISCUSSION THREAD

**Discussion by Rosinantecoffee** [artisan-roaster-scope/artisan#1574] [General]
i bought a victor 86b specifically for the artisan software. Not sure if i have received a faulty item but i keep getting the error "unable to open serial port" which is followed with " SER.HHM28MULTIMETER()@line 6503" and attempting to use a serial port that is not open. if another port is using that COM how do i switch it off so that only artisan is using it?

i have the driver silicon labs cp210x usb to uart bridge installed ( oncom10 tried all baudrate on device manager and on the artisan app) the artisan app device is set to victor 86B - with the same port as the device manager, yet i am still having connection issues... am i missing a key step? or do i have a faulty multimeter ... 

i do have the rs232 button highlighted on the multimeter,

**Comment by poundy** [comment]
I am not a Victor meter user but lets see. You mention device manager so I assume Windows. The trick is to make sure the COM10 port is the correct one by having Device Manager open and the serial port section expanded and the meter disconnected... then connect the cable and confirm that the correct device pops up. 

Then, you should set the port settings on the Device Manager setting to what Artisan expects, which according to the meters page https://artisan-scope.org/devices/meters/ is: 
CP210x driver, 2400-8N1

From there you should then go into artisan and make sure the port settings are exactly the same - 2400,8,N,1.

There's also apparently not much "standardisation" in these meters; if you look at this previous post you can see that if it connects as a USB HID device it isn't supported. https://github.com/artisan-roaster-scope/artisan/issues/861

**Reply by Rosinantecoffee** [reply]
Aha 2400 8 n 1, I was thinking that was another driver name... I will check that setting now and come back (as default the baud rate is set at 9600)..

Apologies I should have mentioned I was using windows 10.

 as for com port setting yeah I do get device manager and artisan to be the same...

Thanks, will report back

**Reply by Rosinantecoffee** [reply]
Lol just amazing, I tried all baudrate except 2400.... it worked 

Thank you
